com.unity.netcode.gameobjects/Runtime/Messaging/NetworkManagerHooks.cs

```diff
@@ -108,6 +111,22 @@ public bool OnVerifyCanReceive(ulong senderId, Type messageType, FastBufferReade
             return !m_NetworkManager.MessageManager.StopProcessing;
         }
 
+        private string GetTransportVersion()
+        {
+            if (NetworkLog.CurrentLogLevel > LogLevel.Normal)
+            {
+                return "";
+            }
+
+            var transportVersion = "NetworkTransport: " + m_NetworkManager.NetworkConfig.NetworkTransport.GetType();
+            if (m_NetworkManager.NetworkConfig.NetworkTransport is UnityTransport a)
```

Comment:
This is pedantic and nitpicky, but could we use a more descriptive variable name here?